Running `make clean` on VS build creates subfolders and messing in multi-build architecture.

On command `make clean`, if there are subfolders that is exist with the Visual Studio project, it will create a mess in cleaning up projects

```bash
(base) sh@DESKTOP-O9NIMTA:/mnt/e/tfslabs/vlmcsd$ make clean
make[1]: Entering directory '/mnt/e/tfslabs/vlmcsd/src'
fatal: No annotated tags can describe '9fc399b8b835579ef64b84e24046e6065ddf035e'.
However, there were unannotated tags: try --tags.
rm -f ../bin/vlmcsd ../bin/vlmcsdmulti ../lib/libkms.so ../bin/vlmcs  ../build/libkms-static.o ../lib/libkms.a ../bin/* ../build/* *.d
rm: cannot remove '../bin/vlmcs-Windows': Is a directory
rm: cannot remove '../bin/vlmcsd-Windows': Is a directory
rm: cannot remove '../bin/vlmcsdmulti-Windows': Is a directory
make[1]: *** [GNUmakefile:655: clean] Error 1
make[1]: Leaving directory '/mnt/e/tfslabs/vlmcsd/src'
make: *** [GNUmakefile:114: clean] Error 2
```

--- a/src/GNUmakefile
+++ b/src/GNUmakefile
@@ -652,10 +652,10 @@ endif
 	$(BUILDCOMMANDPREFIX)$(AR) rcs $@ $^
 
 clean:
-	rm -f $(REAL_PROGRAM_NAME) $(REAL_MULTI_NAME) $(REAL_DLL_NAME) $(REAL_CLIENT_NAME)  $(OBJ_NAME) $(REAL_A_NAME) ../bin/* ../build/* *.d
+	rm -rf $(REAL_PROGRAM_NAME) $(REAL_MULTI_NAME) $(REAL_DLL_NAME) $(REAL_CLIENT_NAME)  $(OBJ_NAME) $(REAL_A_NAME) ../bin/* ../build/* *.d
 
 dnsclean:
-	rm -f ../build/dns_srv.o
+	rm -rf ../build/dns_srv.o
 
 help:
 	@echo "Help is available by typing 'make help' in directory $(shell realpath `pwd`/..). Use 'cd ..' to get there."

--- a/src/GNUmakefile
+++ b/src/GNUmakefile
@@ -652,7 +652,7 @@ endif
 	$(BUILDCOMMANDPREFIX)$(AR) rcs $@ $^
 
 clean:
-	rm -rf $(REAL_PROGRAM_NAME) $(REAL_MULTI_NAME) $(REAL_DLL_NAME) $(REAL_CLIENT_NAME)  $(OBJ_NAME) $(REAL_A_NAME) ../bin/* ../build/* *.d
+	rm -rf $(REAL_PROGRAM_NAME) $(REAL_MULTI_NAME) $(REAL_DLL_NAME) $(REAL_CLIENT_NAME) $(OBJ_NAME) $(REAL_A_NAME) ../bin/* ../build/* *.d
 
 dnsclean:
 	rm -rf ../build/dns_srv.o

